### ISSUE TITLE
Example for change_e fails

### ISSUE THREAD

**Issue by iagomosqueira** [ss3sim/ss3sim#243]
I was trying to run the example for change_e and got an error on the path being created to access the control file. After the first few lines of the example I get to

```
> change_e(ctl_file_in = ctl_file, ctl_file_out = "change_e.ctl",
+          dat_list = data.old, for_file_in = "forecast.ss",
+          natM_type = "n_breakpoints", natM_n_breakpoints = c(1, 4),
+          natM_lorenzen = NULL, natM_val = c(.2, 3, 0.4, 5),
+          par_name = c("_steep", "SizeSel_1P_1_Fishery"),
+          par_int = c(0.3, 40), par_phase = c(3, 2),
+          forecast_num = 0, run_change_e_full = TRUE )

Error in file(con, "r") : cannot open the connection
In addition: Warning message:
In file(con, "r") :
  cannot open file '/tmp/RtmpfkiXro/ss3sim-tv-example//home/mosquia/R/x86_64-pc-linux-gnu-library/3.4/ss3sim/extdata/models/cod-om/codOM.ctl': No such file or directory
```

the tempdir path (`/tmp/RtmpfkiXro/ss3sim-tv-example/`) and the path of the ctl file inside the ss3sim/ext-data folder (`/home/mosquia/[...]/ss3sim/extdata/models/cod-om/codOM.ctl`) appear to be merged somehow.

This is using the github master in Linux, with R 3.4.

**Comment by kellijohnson** [comment]
@iagomosqueira thank you for noticing this. r4ss/r4ss@6f93e23b55a7d9b33d2ffbb95dc909e07bed4bb7 is pasting the working directory to the file path b/c we have `dir = NULL`. I will have to fix this inside `change_e`.

**Comment by kellijohnson-NOAA** [comment]
This is now fixed with 2df6f93 using basename() and dirname() after copying over the in file to be the out file.
